src/Api/Startup.cs

```diff
@@ -66,6 +68,15 @@ public void ConfigureServices(IServiceCollection services)
             services.Configure<IpRateLimitPolicies>(Configuration.GetSection("IpRateLimitPolicies"));
         }
 
+        // TODO: Be more selective about adding this scaffolder
+        if (Environment.IsDevelopment())
```

Comment:
We likely want to be much more strict on when this runs. Some additional checks I am considering:

1. Have this be behind a `DEBUG` compiler directive
2. Only run this when ALL azure related connection strings are `UseDevelopmentStorage=true`.
3. Only run this when ANY azure related connection strings are `UseDevelopmentStorage=true`. 
4. Run very selective scaffolding only for the things choosing `UseDevelopmentStorage=true`. 
5. `#1 and #2`
6. `#1 and #3`